patterns/nagios

```diff
@@ -55,8 +55,13 @@ NAGIOS_EC_DISABLE_HOST_CHECK DISABLE_HOST_CHECK
 NAGIOS_EC_ENABLE_HOST_CHECK ENABLE_HOST_CHECK
 NAGIOS_EC_PROCESS_SERVICE_CHECK_RESULT PROCESS_SERVICE_CHECK_RESULT
 NAGIOS_EC_PROCESS_HOST_CHECK_RESULT PROCESS_HOST_CHECK_RESULT
-NAGIOS_EC_SCHEDULE_SERVICE_DOWNTIME SCHEDULE_SERVICE_DOWNTIME
+NAGIOS_EC_SCHEDULE_SVC_DOWNTIME SCHEDULE_SVC_DOWNTIME
```

Comment:
Also the "NAGIOS_EC_SCHEDULE_SERVICE_DOWNTIME" pattern was never actually used further down in the file, so no regressions are likely.